kinder/ci/workflows/discovery-master.yaml

```diff
@@ -0,0 +1,147 @@
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used for testing alternative
+  discovery methods for kubeadm join.
+vars:
+  kubernetesVersion: tmp/v1.16.0-alpha.2.128+86ebaaa0232999 #"{{ resolve `ci/latest` }}"
```

Comment:
why not use the "ci/latest" label; the latest fix changes are already in master?